tips/templates/tips/categories.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %} {%load static%} {%block content%}
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML 5 Boilerplate</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <script src="index.js"></script>
+    {% for tip in category_filtered_tips %}
+        <article class="post vt-post">
+            <div class="row">
+                <div class="col-xs-12 col-sm-5 col-md-5 col-lg-4">
+                    <div class="post-type post-img">
+                        <a href="#"><img src="https://bootdey.com/img/Content/avatar/avatar1.png" class="img-responsive" alt="image post"></a>
+                    </div>
+                    <div class="author-info author-info-2">
+                        <ul class="list-inline">
+                            <li>
+                                <div class="info">
+                                    <p>Posted on:</p>
+                                    <strong>{{ tip.date}}</strong></div>
+                            </li>
+                            <li>
+                                <div class="info">
+                                    <p>Category:</p>
+                                    <strong>{{ tip.category }}</strong></div>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="col-xs-12 col-sm-7 col-md-7 col-lg-8">
+                    <div class="caption">
+                        <h3 class="md-heading"><a href="#">{{ tip.author }}</a></h3>
+                        <p> {{ tip.text }}</p>
+                        <a class="btn btn-default" href="#" role="button">Read More</a> </div>
```

Comment:
What the href represents?

Reply:
right now it is sending the user to top of the page after if we want we could add more functuionalitis like going to author profile